fairseq2n/python/src/fairseq2n/bindings/data/data_pipeline.cc

```diff
@@ -358,8 +358,35 @@ def_data_pipeline(py::module_ &data_module)
                 return data_pipeline::count(start, std::move(key));
             },
             py::arg("start") = 0,
-            py::arg("key") = std::nullopt);
+            py::arg("key") = std::nullopt)
+        .def_static(
+            "cat",
```

Comment:
I would prefer to name this op `concat` to make it a bit more descriptive.

Reply:
Ok I will change it